restaurants/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.11.0b1-slim-buster
```

Comment:
will maybe change this dockerfiles functionality but it can be used as a placeholder file for time being as rest of pr has useful code and funtions